osc-staging.py

```diff
@@ -157,6 +158,11 @@ def do_staging(self, subcmd, opts, *args):
         changed from state new or review more than 3 days ago will be removed.
 
     "list" will list/supersede requests for ring packages or all if no rings.
+        As an example, the following would list all packages which have received a
```

Comment:
I think we should make clear that this is not necessary (as `list` works fine without `--match-filter`).

Reply:
thanks. fixed